src/grafanarmadillo/cmd.py

```diff
@@ -0,0 +1,173 @@
+"""Ready-to-run commands for common Grafana templating scenarios."""
+import json
+from pathlib import Path
+from typing import Dict, Literal, NewType, Union
+
+import click
+from grafana_client import GrafanaApi
+
+from grafanarmadillo.alerter import Alerter
+from grafanarmadillo.dashboarder import Dashboarder
+from grafanarmadillo.find import Finder
+from grafanarmadillo.templator import Templator, findreplace
+
+
+EnvMapping = NewType("EnvMapping", Dict[str, Dict[str, str]])
+Direction = NewType("Direction", Union[Literal["import"], Literal["export"]])
+
+load_file_help = """Should be encoded as json. You can pass this in as a string; or as file using 'file://path/to/file'"""
+mapping_help = \
+	"Mapping of values to findreplace in the template. " + \
+	"The type should be a mapping of environment names (like 'template' or 'production') " + \
+	"to dicts of replacements (for example, `{\"region\": \"SouthWest\"}`). " + \
+	load_file_help
+env_grafana_help = "Name of the environment in the mapping file for Grafana, found in the `--mapping` argument"
+env_template_help = "Name of the environment in the mapping file for the template, found in the `--mapping` argument"
+
+
+def load_data(data_str: str):
+	"""Attempt to load data."""
+	_file_uri_prefix = "file://"
+	if data_str.startswith(_file_uri_prefix):
+		filename = Path(data_str.split(_file_uri_prefix)[1])
+		with filename.open(mode="r", encoding="utf-8") as data_file:
+			return json.load(data_file)
+	else:
+		return json.loads(data_str)
+
+
+def make_mapping_templator(mapping: EnvMapping, env_grafana: str, env_template: str) -> Templator:
+	"""Assemble the templator from the environment mapping."""
+	mapping_grafana = mapping[env_grafana]
+	mapping_template = mapping[env_template]
+
+	# if some keys in the src mapping are not in the dst mapping
+	missing = set(mapping_grafana.keys()) - set(mapping_template.keys())
+	if missing:
+		raise ValueError(f"Some keys in the source mapping are not present in the destination mapping. {missing=}")
+
+	grafana_to_template = {}
+	for k, v in mapping_grafana.items():
+		grafana_to_template[v] = mapping_template[k]
```

Comment:
```suggestion
	grafana_to_template = {value: mapping_template[key] for key, value in mapping_grafana.items()}
```
worth just throwing as a one-liner?